gateleen-routing/src/main/java/org/swisspush/gateleen/routing/Router.java

```diff
@@ -344,7 +342,9 @@ private void createForwarders(List<Rule> rules, io.vertx.ext.web.Router newRoute
             } else if (rule.getScheme().equals("local")) {
                 forwarder = new Forwarder(vertx, selfClient, rule, this.storage, loggingResourceManager, monitoringHandler, userProfileUri);
             } else {
+                HttpClient client = vertx.createHttpClient( rule.buildHttpClientOptions() );
```

Comment:
interesting finding - we really created HttpClients for "local" routes and didn't recognized this before?

Reply:
I found this during #239 (See [diff](https://github.com/swisspush/gateleen/pull/239/files#diff-8fa3f5fa721d37d2dad6ccd9edde2d7cR333)) (do you remind?). Back then I didn't change that because I didn't know about possible side effects and added that TODO instead.